### ISSUE TITLE
Upgrading package from 4.12.0 to 4.13.0

### ISSUE THREAD

**Issue by sawilde** [getsentry/sentry-symfony#791]
### How do you use Sentry?

Sentry SaaS (sentry.io)

### SDK version

4.13.0

### Steps to reproduce

symfony/symfony (4.4.51) - we are planning on upgrading but it is not a tomorrow thing
sentry/sentry-symfony (4.12.0)

upgrade to sentry/sentry-symfony (4.13.0)

```
Package operations: 0 installs, 2 updates, 0 removals
  - Downloading sentry/sentry-symfony (4.13.0)
  - Upgrading sentry/sdk (3.5.0 => 3.6.0)
  - Upgrading sentry/sentry-symfony (4.12.0 => 4.13.0): Extracting archive
```

...
```
In DecoratorServicePass.php line 83:
                                                                                                                                      
  The service "Sentry\SentryBundle\Tracing\HttpClient\TraceableHttpClient" has a dependency on a non-existent service "http_client".  
```

Not something I would've expected for a minor package update

### Expected result

Upgrade works without any code change or the notes within the upgrade contain want needs to be done to take the new version

### Actual result

```
In DecoratorServicePass.php line 83:
                                                                                                                                      
  The service "Sentry\SentryBundle\Tracing\HttpClient\TraceableHttpClient" has a dependency on a non-existent service "http_client".  
```

**Comment by ste93cry** [comment]
I submitted a PR to address the issue. Would you mind checking if it works?

**Comment by cleptric** [comment]
Fixed in [4.13.1](https://github.com/getsentry/sentry-symfony/releases/tag/4.13.1)
